### DISCUSSION TITLE
Mind Elixir Icons

### DISCUSSION THREAD

**Discussion by DrSamy** [SSShooter/mind-elixir-core#284] [General]
Hi,

Using Mind Elixir fullscreen, I wasnt able to shrink the screen back to normal by clicking on the icon.
I want to propose a fix with an appropriate icon. Add a new svg in iconfont.js seems appropriate, but I was wondering which library is used and how to implement a new icon ?

Regards

**Comment by SSShooter** [comment]
It seems very difficult to extend iconfont.js because it's generated on https://www.iconfont.cn/

If your project needs a shrink button, you can just inject the button into the fullscreened DOM 🤔
